build.rs

```diff
@@ -1,17 +0,0 @@
-extern crate vergen;
-
-use vergen::{ConstantsFlags, Result, Vergen};
-
-fn main() {
-    gen_constants().expect("Unable to generate vergen constants!");
-}
-
-fn gen_constants() -> Result<()> {
-    let vergen = Vergen::new(ConstantsFlags::all())?;
-
-    for (k, v) in vergen.build_info() {
-        println!("cargo:rustc-env={}={}", k.name(), v);
-    }
-
-    Ok(())
-}
```

Comment:
Hmm I'll have to try and recall what the deal with this was, but I'd like to avoid just deleting this...